cli/flow-typed/jest.js.flow

```diff
@@ -3,10 +3,55 @@ declare function describe(suiteName: string, cb: () => void): void;
 declare function expect(value: any): Object;
 declare function it(testName: string, cb: (done: () => void) => void): void;
 declare function pit(testName: string, cb: () => Promise<void>): void;
+
+type MockFn = {
```

Comment:
nit: Would it make sense to call this `JestMockFn` (since this is a global and all)?